src/rsa_api/rsa_api.py

```diff
@@ -2406,6 +2412,8 @@ def IQSTREAM_StatusParser(
             if bool(status & 0x200000):  # mask bit 21
                 status_str += "Output buffer overflow. File writing too slow, "
                 status_str += "data loss has occurred.\n"
+            if status_str == "":
```

Comment:
It seems like if there is a known error and unknown error, only the known error will be reported. Should it be changed such that unknown error will also be reported?

Reply:
This is now fixed